docs/eigenda/operator-guides/requirements/system-requirements.md

```diff
@@ -0,0 +1,62 @@
+---
+sidebar_position: 1
+---
+
+# System Requirements
+
+## General System Requirements
+
+The EigenDA network design dictates that operators with greater stake will
+receive more frequent chunk dispersal requests, since they operate with greater
+cryptoeconomic security. This means that an operator's node requirements are a
+function of the total amount of stake they wield across all quorums, which we
+call 'Total Quorum Stake' (TQS). For example, if an operator Foobar has 3% stake
+on the restaked ETH quorum, and 5% ETH on a staked WETH quorum, then operator
+Foobar's TQS is 8%.
+
+Operators should use the following table to determine which [EigenLayer node class](/eigenlayer/operator-guides/eigenlayer-node-classes#general-purpose-eigenlayer-node-classes)
+is appropriate for their level of stake:
+
+| Total Quorum Stake (TQS) |                         | Supported Throughput |
```

Comment:
may clarify if "Supported Throughput" means min throughput they should support, or the max throughput they need to support